pkg/config/filter.go

```diff
@@ -67,7 +88,13 @@ func (f Filter) Match(o string) (bool, error) {
 		}
 		fieldTime, err := parseDate(o)
 		if err != nil {
-			return false, nil
+			var ferr = &FilterFailed{
+				Filter: FilterTypeDateOlderThan,
+				Fatal: false,
+				Err: err,
+			}
+
+			return false, ferr
```

Comment:
An example of how to implement the filterfailed to pass more information to the caller.  
In this situation the dateOlderThan filter encountered an error (the date could not be passed from the tag provided), this error causes the filter to be unable to provide an accurate answer, but does not impair the program from continuing.  
Therefor the information provided is: This is who I am, this is not a mortal wound and this is what caused it.